src/main/java/frc/robot/Robot.java

```diff
@@ -50,7 +50,9 @@ public void robotPeriodic() {
 
   /** This function is called once each time the robot enters Disabled mode. */
   @Override
-  public void disabledInit() {}
+  public void disabledInit() {
+    m_robotContainer.coastDrive();
```

Comment:
lol is this so you can push the robot?

Reply:
Yeahhh, I'm sure there's a prettier solution, but this works